website/versioned_docs/version-v2.0_alpha/developers-references/smart-contracts/Poll.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Poll Smart Contract
+description: MACI Poll Contract
+sidebar_label: Poll
+sidebar_position: 2
+---
+
+This contract allows users to submit their votes.
+
+The main functions of the contract are as follows:
+
+- `publishMessage` - This function allows anyone to publish a message, and it accepts the message object as well as an ephemeral public key. This key together with the coordinator public key will be used to generate a shared ECDH key that will encrypt the message.
+  Before saving the message, the function will check that the voting deadline has not passed, as well as the max number of messages was not reached.
+- `publisMessageBatch` - This function allows to submit a batch of messages, and it accepts an array of messages with their corresponding public keys used in the encryption step. It will call the `publishMessage` function for each message in the array.
+
+The `publishMessage` function looks as follows:
```

Comment:
same as the MACI contract, since there are several functions which might be mentioned in other page, maybe could add subtitle to them.